internal/report/junit/junit.go

```diff
@@ -174,8 +175,8 @@ func (r *Reporter) Render() {
 	_, _ = fmt.Fprint(f, "\n")
```

Comment:
I noticed that the errors from `f.Write()` and `fmt.Fprint()` are being ignored. While these errors are rare, shouldn't they be handled?

Reply:
Yup, they should have been. I have some more refactors coming up, where I'll include those changes 👍